### ISSUE TITLE
Ernie-Doc使用linear_warmup_decay学习率一直为0

### ISSUE THREAD

**Issue by jy1993** [PaddlePaddle/ERNIE#683]
运行配置如下：
is_zh="True"
repeat_input="False"
train_all="Fasle"
eval_all="False"
use_vars="False"
use_amp="False"
use_recompute="False"
rel_pos_params_sharing="False"
lr_scheduler="linear_warmup_decay"
vocab_path="./configs/base/zh/vocab.txt"
config_path="./configs/base/zh/ernie_config.json"
init_model_checkpoint="model-ernie-doc-base-zh"
init_model_pretraining=""

epoch=1
warmup=0.1
max_len=512
lr_rate=7e-5
batch_size=16
weight_decay=0.01
num_labels=2
save_steps=5000
validation_steps=5000
layer_decay_ratio=0.8

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/18455572/119762571-8ba91d80-bee0-11eb-91e5-2757e9e1ac14.png">

**Comment by sjy1203** [comment]
您好，finetune时候用init_model_pretraining=""来加载预训练模型

**Comment by jy1993** [comment]
多谢，请教一下init_model_checkpoint是用来干嘛的

**Comment by sjy1203** [comment]
init_model_checkpoint这个配置项是做预训练用

**Comment by jy1993** [comment]
好的，谢谢

**Comment by wangddddddd** [comment]
> 您好，finetune时候用init_model_pretraining=""来加载预训练模型
您好，还是不能解决，还是一直为零，是什么情况呢

**Comment by wangddddddd** [comment]
> 运行配置如下：
> is_zh="True"
> repeat_input="False"
> train_all="Fasle"
> eval_all="False"
> use_vars="False"
> use_amp="False"
> use_recompute="False"
> rel_pos_params_sharing="False"
> lr_scheduler="linear_warmup_decay"
> vocab_path="./configs/base/zh/vocab.txt"
> config_path="./configs/base/zh/ernie_config.json"
> init_model_checkpoint="model-ernie-doc-base-zh"
> init_model_pretraining=""
> 
> epoch=1
> warmup=0.1
> max_len=512
> lr_rate=7e-5
> batch_size=16
> weight_decay=0.01
> num_labels=2
> save_steps=5000
> validation_steps=5000
> layer_decay_ratio=0.8
> 
> <img alt="image" width="1200" src="https://user-images.githubusercontent.com/18455572/119762571-8ba91d80-bee0-11eb-91e5-2757e9e1ac14.png">

您好，楼主，请问后来这个问题怎么解决的呢？
